app/src/main/java/ch/admin/bag/dp3t/whattodo/WtdPositiveTestFragment.java

```diff
@@ -47,10 +55,70 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
 			UrlUtil.openUrl(getContext(), getString(R.string.faq_button_url));
 		});
 
-		view.findViewById(R.id.wtd_inform_call_infoline_coronavirus).setOnClickListener(v->{
-			PhoneUtil.callInfolineCoronavirus(v.getContext());
-		});
+		View oldCallButton = view.findViewById(R.id.wtd_inform_call_infoline_coronavirus);
+		if (oldCallButton != null) {
+			oldCallButton.setOnClickListener(v -> {
+				PhoneUtil.callInfolineCoronavirus(v.getContext());
+			});
+		}
+	}
+
+	private void fillContentFromConfigServer(View view) {
+		Context context = view.getContext();
+		SecureStorage secureStorage = SecureStorage.getInstance(context);
+		WhatToDoPositiveTestTextsModel textModel =
+				secureStorage.getWhatToDoPositiveTestTexts(context.getString(R.string.language_key));
+
+		if (textModel != null) {
```

Comment:
Shouldn't the Views get hidden if there's no data? Right now we'd have just some empty Views.

Reply:
nevermind